app/cypress/integration/email.spec.js

```diff
@@ -0,0 +1,146 @@
+if (Cypress.env('NO_MAIL')) {
+  describe('No tests run', () => {
+    it('NO_MAIL flag was set', () => {
+      cy.log('NOOP');
+    });
+  });
+} else {
+  describe('When a user is created', () => {
+    before(() => {
+      cy.sqlFixture('fixtures/add-user');
+    });
+    after(() => {
+      cy.sqlFixture('fixtures/remove-user');
+    });
+
+    it('should send an automatic email upon registration', () => {
+      cy.wait(500);
+      cy.request('localhost:8025/api/v1/messages').then(response => {
+        // eslint-disable-next-line jest/valid-expect
+        expect(response.status).to.eq(200);
+        // eslint-disable-next-line jest/valid-expect
+        expect(response.body[0].Content.Body).to.contain(
+          'Thank you for registering'
+        );
+        cy.request('DELETE', 'localhost:8025/api/v1/messages');
+      });
+    });
+  });
+
+  describe('Certification & Confirmation emails', () => {
+    before(() => {
+      cy.sqlFixture('fixtures/email-setup');
+    });
+    beforeEach(() => {
+      cy.logout();
+      cy.login(
+        Cypress.env('TEST_REPORTER_USERNAME'),
+        Cypress.env('TEST_REPORTER_PASSWORD')
+      );
+    });
+    after(() => {
+      cy.wait(1000);
+      cy.sqlFixture('fixtures/email-teardown');
+    });
+    it('should send notification emails to the certifier when the reporter sends a request, and to the reporter if the data is out of date', () => {
+      const applicationId = window.btoa('["applications", 2]');
+      cy.visit(
+        `/reporter/ciip-application?applicationId=${applicationId}&confirmationPage=true&version=1`
+      );
+      cy.url().should('include', '/reporter/ciip-application');
+      cy.get('#certifierEmail')
+        .clear()
+        .type('certifier@certi.fy');
+      cy.get('.btn')
+        .contains('Send to Certifier')
+        .click();
+      cy.wait(1000);
+      cy.request('localhost:8025/api/v1/messages').then(response => {
+        // eslint-disable-next-line jest/valid-expect
+        expect(response.status).to.eq(200);
+        // eslint-disable-next-line jest/valid-expect
+        expect(response.body[0].To[0].Mailbox).to.contain('certifier');
+        // eslint-disable-next-line jest/valid-expect
```

Comment:
Why do you have to disable this rule?

Reply:
@matthieu-foucault It doesn't like some expressions like `to.contain`, it complains it isn't valid, but this is what I found to work. I think it's using mocha/chai expressions but eslint is linting it for jest?